routers/web/user/notification.go

```diff
@@ -169,56 +169,56 @@ func NotificationStatusPost(c *context.Context) {
 	case "pinned":
 		status = activities_model.NotificationStatusPinned
 	default:
-		c.ServerError("InvalidNotificationStatus", errors.New("Invalid notification status"))
+		ctx.ServerError("InvalidNotificationStatus", errors.New("Invalid notification status"))
 		return
 	}
 
-	if _, err := activities_model.SetNotificationStatus(notificationID, c.Doer, status); err != nil {
-		c.ServerError("SetNotificationStatus", err)
+	if _, err := activities_model.SetNotificationStatus(ctx, notificationID, ctx.Doer, status); err != nil {
+		ctx.ServerError("SetNotificationStatus", err)
 		return
 	}
 
-	if !c.FormBool("noredirect") {
-		url := fmt.Sprintf("%s/notifications?page=%s", setting.AppSubURL, url.QueryEscape(c.FormString("page")))
-		c.Redirect(url, http.StatusSeeOther)
+	if !ctx.FormBool("noredirect") {
+		url := fmt.Sprintf("%s/notifications?page=%s", setting.AppSubURL, url.QueryEscape(ctx.FormString("page")))
+		ctx.Redirect(url, http.StatusSeeOther)
 	}
 
-	getNotifications(c)
-	if c.Written() {
+	getNotifications(ctx)
+	if ctx.Written() {
 		return
 	}
-	c.Data["Link"] = setting.AppURL + "notifications"
-	c.Data["SequenceNumber"] = c.Req.PostFormValue("sequence-number")
+	ctx.Data["Link"] = setting.AppURL + "notifications"
+	ctx.Data["SequenceNumber"] = ctx.Req.PostFormValue("sequence-number")
 
-	c.HTML(http.StatusOK, tplNotificationDiv)
+	ctx.HTML(http.StatusOK, tplNotificationDiv)
 }
 
 // NotificationPurgePost is a route for 'purging' the list of notifications - marking all unread as read
-func NotificationPurgePost(c *context.Context) {
-	err := activities_model.UpdateNotificationStatuses(c.Doer, activities_model.NotificationStatusUnread, activities_model.NotificationStatusRead)
+func NotificationPurgePost(ctx *context.Context) {
+	err := activities_model.UpdateNotificationStatuses(ctx.Doer, activities_model.NotificationStatusUnread, activities_model.NotificationStatusRead) // TODO
```

Comment:
That was for me to pass the parameter into `UpdateNotificationStatuses`. I switched to a different `// TODO` comment later because there are so many todo comments I couldn't find my own comments anymore, like in this case. 😏